sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/GetLoginBehaviourRisk.java

```diff
@@ -0,0 +1,177 @@
+import org.wso2.extension.siddhi.execution.geovelocity.internal.impl.GeoVelocityDataResolverHolder;
+import org.wso2.siddhi.annotation.Example;
+import org.wso2.siddhi.annotation.Extension;
+import org.wso2.siddhi.annotation.Parameter;
+import org.wso2.siddhi.annotation.ReturnAttribute;
+import org.wso2.siddhi.annotation.util.DataType;
+import org.wso2.siddhi.core.config.SiddhiAppContext;
+import org.wso2.siddhi.core.executor.ExpressionExecutor;
+import org.wso2.siddhi.core.executor.function.FunctionExecutor;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+import org.wso2.siddhi.query.api.definition.Attribute;
+import org.wso2.siddhi.query.api.exception.SiddhiAppValidationException;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class is to get the risk of login based on
+ * the user's login behaviour.
+ */
+@Extension(
+        name = "loginBehaviourBasedRisk",
+        namespace = "geovelocity",
+        description = "Returns the calculated login behaviour based risk" +
+                "considering the location and the time of the login",
+        parameters = {
+                @Parameter(
+                        name = "username",
+                        description = "User's username",
+                        type = {DataType.STRING}),
+                @Parameter(
+                        name = "city",
+                        description = "User's current login city",
+                        type = {DataType.STRING}),
+                @Parameter(
+                        name = "currentlogintime",
+                        description = "timestamp of current login",
+                        type = {DataType.LONG})
+        },
+        returnAttributes =
+        @ReturnAttribute(
+                description = "Returns the login behaviour based calculated risk" +
+                        "considering the location and the time of the login." +
+                        "The range of the risk value is 0 to 1.",
+                type = {DataType.DOUBLE}),
+        examples = @Example(
+                description = "This will return the login behaviour based risk " +
+                        "considering the location and the time of the login. " +
+                        "Assume Alex last login from New York was yesterday, and he tries to " +
+                        "login to the system today, then his login Behaviour based risk is low. But if he tries to " +
+                        "login to the system 1 year back, then his login behaviour is high. ",
+                syntax = "define stream GeovelocityStream(username string, city string," +
+                        "currentlogintime string);\n" +
+                        "from GeovelocityStream\n" +
+                        "select geovelocity:loginBehaviourBasedRisk(username, city," +
+                        "currentlogintime) as loginbehaviourbasedrisk \n" +
+                        "insert into outputStream;")
+)
+
```

Comment:
remove new line

Reply:
Fixed in https://github.com/wso2/samples-is/pull/265